webpack.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
 		]
 	}
 	, resolve: {
-		extensions: ['', '.js', '.jsx']
+		extensions: ['.js', '.jsx']
```

Comment:
This should not be removed. An empty string here allows extensions based on the extension specified when the file is required.

Here's the quote from the webpack documentation:

`If you want modules that were required with their extension (e.g. require('./somefile.ext')) to be properly resolved, you must include an empty string in your array. Similarly, if you want modules that were required without extensions (e.g. require('underscore')) to be resolved to files with ".js" extensions, you must include ".js" in your array.`